tox.ini

```diff
@@ -1,6 +1,71 @@
+[tox]
+skipsdist=True
+envlist = cov, flake8
+
+[testenv]
+basepython = python3
+deps = -rrequirements.txt
+
+
+[testenv:rep]
+commands =
+    pytest --cov=app --cov-report=html
+
+
+[testenv:cov]
+commands =
+    pytest --cov=app
+
+
+[testenv:flake8]
+deps = flake8
+commands =
+    flake8 app
+    flake8 tests
+
+
+# pytest Configuration
+[pytest]
+junit_family = xunit2
+markers =
```

Comment:
We can't allow this - I don't want a coupling that should be added every time we add a new testfile